docker-compose.travis-cmis.yml

```
@@ -11,7 +11,7 @@ version: "2"
 services:
     alfresco:
         image: alfresco/alfresco-content-repository-community:6.1.2-ga
-        mem_limit: 1500m
+        mem_limit: 2500m
```

Looking at the Travis [build environments](https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system), it looks like the VMs have 7.5GB of memory.

I think it wouldn't hurt to just make this 4GB for Alfresco, then there's still 3.5GB left over for the rest which should be plenty.

```suggestion
        mem_limit: 4g
```